sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -82,13 +90,13 @@
         this.completionMono = renewLockOperation.then();
         this.subscription = renewLockOperation.subscribe(until -> this.lockedUntil.set(until),
             error -> {
-                logger.error("token[{}]. Error occurred while renewing lock token.", error);
+                logger.error("Error occurred while renewing lock token.", error);
```

Comment:
I wonder if it is necessary to log the lockToken that failed in this case and the one below?

Reply:
we still log it on all records here via global context (set in constructor)